### PR TITLE
Remove references to service bus

### PR DESCRIPTION
**Pull request by rkondratowicz**

### CODE REVIEW

**Comment by banderous** [comment]
Looks good to me!

File: src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfMerger.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package uk.gov.hmcts.reform.slc.services.steps.getpdf;
 
 import org.apache.pdfbox.multipdf.PDFMergerUtility;
-import uk.gov.hmcts.reform.slc.services.servicebus.exceptions.PdfMergeException;
+import uk.gov.hmcts.reform.slc.services.steps.getpdf.exceptions.PdfMergeException;
```
Comment (KarolJastrzebski) [inline src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfMerger.java]: Would renaming `steps.getpdf` to `pdf` make sense?

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfMerger.java]: 👏 . i'd leave for refactor anyway. at this time we are in the position to produce the quickest merge possible. seeing other stuff in the source code which ought to be amended/removed

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfMerger.java]: just removing service bus references here

**Pull request merged**